src/helpers/BoxHelper.js

```diff
@@ -83,7 +84,7 @@ BoxHelper.prototype.update = ( function () {
 
 		position.needsUpdate = true;
 
-		this.geometry.computeBoundingSphere();
+		return this;
```

Comment:
It is not clear to me the usefulness of returning `this` from the `update()` method. Again, whatever we do, it should be consistent.